src/Paket.Core/Paket.Core.fsproj

```diff
@@ -63,6 +63,11 @@
         <FSharpTargetsPath>$(MSBuildProgramFiles32)\Microsoft SDKs\F#\4.0\Framework\v4.0\Microsoft.FSharp.Targets</FSharpTargetsPath>
       </PropertyGroup>
     </When>
+    <When Condition="$(TargetFSharpCoreVersion) &gt;= 4.4.1.0 AND $(TargetFSharpCoreVersion) &lt; 4.4.1.0 ">
```

Comment:
The target F# core version is a lot less useful when you're actually relying on paket and the fsharp.core nupkg to determine the version you're using